src/application.ts

```diff
@@ -32,6 +35,7 @@ export default class Application implements Owner {
   public rootElement: any;
   public resolver: Resolver;
   public env: Environment;
+  protected roots: object[] = [];
```

Comment:
Let's also add an `id: number` to it for the `#each`. We can just keep a counter and increment it every time you push something